# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fuse-emulator/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

# T2 SDE: package/.../fuseemulator/gcc4.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- fuse-0.7.0/sound.c_orig	2006-04-10 20:02:40.000000000 +0100
+++ fuse-0.7.0/sound.c	2006-04-10 20:02:49.000000000 +0100
@@ -58,8 +58,8 @@
 int sound_stereo_ay_abc=0;	/* (AY stereo) true for ABC stereo, else ACB */
 int sound_stereo_ay_narrow=0;	/* (AY stereo) true for narrow AY st. sep. */
 
-static int sound_stereo_ay=0;	/* local copy of settings_current.stereo_ay */
-static int sound_stereo_beeper=0;   /* and settings_current.stereo_beeper */
+int sound_stereo_ay=0;	/* local copy of settings_current.stereo_ay */
+int sound_stereo_beeper=0;   /* and settings_current.stereo_beeper */
 
 
 /* assume all three tone channels together match the beeper volume (ish).
